modules/nf-core/amulety/translate/main.nf

```diff
@@ -0,0 +1,47 @@
+process AMULETY_TRANSLATE {
+    tag "$meta.id"
+    label 'process_single'
+
+    conda "${moduleDir}/environment.yml"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/mulled-v2-92ebbfc09fc136b8e201cb187cd9567ba335d439:459e6ebe51fb2818cb6de807f2c5fa99599b1214-0':
+        'biocontainers/mulled-v2-92ebbfc09fc136b8e201cb187cd9567ba335d439:459e6ebe51fb2818cb6de807f2c5fa99599b1214-0' }"
+
+    input:
+    tuple val(meta), path(tsv)
+    path(reference_igblast) // igblast references
+
+    output:
+    tuple val(meta), path("*_translated.tsv") , emit: repertoire_translated
+    path "versions.yml" , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+    def prefix = task.ext.prefix ?: "${meta.id}"
+    """
+    export IGDATA=${reference_igblast}
+    amulety translate-igblast $tsv . ${reference_igblast}
```

Comment:
You could add `mv *_translated.tsv ${prefix}_translated.tsv` if you want, to use the prefix variable both in the stub, and in the main script output.

Reply:
Good point, I'll do that!